README.md

```diff
@@ -152,7 +152,7 @@ For feature requests, bugs, or technical questions, email us at [dev@buf.build][
 [protobuf]: https://developers.google.com/protocol-buffers
 [pushing]: https://docs.buf.build/bsr/usage#push-a-module
 [releases]: https://docs.buf.build/installation#github-releases
-[repo]: ./
+[repo]: https://github.com/bufbuild/buf/
```

Comment:
Otherwise, we link to https://github.com/bufbuild/buf/blob/main, which is a 404.